templates/model.txt

```diff
@@ -1,6 +1,10 @@
+// This model was generated by Lumber. However, you remain in control of your models.
+// Learn how to enrich this model here: https://docs.forestadmin.com/documentation/v/v4/reference-guide/models/enrich-your-models
 <% if (dialect === 'mongodb') {
 %>const mongoose = require('mongoose');
 
+// This section contains properties mapping.
```

Comment:
This sections contains the properties of your model, mapped to your collection's properties. Learn more here: https://docs.forestadmin.com/documentation/v/v4/reference-guide/models/enrich-your-models#declaring-a-new-field-in-a-model